system/protocols/mumble/protocol.py

```diff
@@ -582,37 +611,75 @@ def handle_msg_userstate(self, message):
         if message.name and message.session not in self.users:
             # Note: I'm not sure if message.name should ever be empty and
             # not in self.users - rakiru
-            self.users[message.session] = User(self,
-                                               message.session,
-                                               message.name,
-                                               self.channels[
-                                                   message.channel_id],
-                                               message.mute,
-                                               message.deaf,
-                                               message.suppress,
-                                               message.self_mute,
-                                               message.self_deaf,
-                                               message.priority_speaker,
-                                               message.recording)
+            # Note: RE: above note - Mumble desktop client source suggests not.
+            user = User(self,
```

Comment:
That's a fair point, didn't consider reformatting. As you've seen,  I'm not a fan of multiple attrs per line if they span multiple lines (looks weird, but also messes with git when line wrapping changes). If anything, I'd prefer to put the first and last ones on their own lines too (rather than with parentheses). Hmm...

            user = User(self, message.session, message.name,
                        self.channels[message.channel_id], message.mute,
                        message.deaf, message.suppress, message.self_mute,
                        message.self_deaf, message.priority_speaker,
                        message.recording)
            self.users[message.session] = user